src/index.d.ts

```diff
@@ -1729,6 +1729,11 @@ export interface SocialIconProps {
    */
   button?: boolean;
 
+  /**
+   * Social media name
+   */
+  name: SocialMediaType;
```

Comment:
This isn't correct, this is supposed to be like `facebook` `google` etc

Reply:
I just exported strings into variable called [`SocialMediaType`](https://github.com/react-native-training/react-native-elements/blob/8fedf7906c7d235844917cb2d101836d04e46794/src/index.d.ts#L1684) just like [`IconType`](https://github.com/react-native-training/react-native-elements/blob/8fedf7906c7d235844917cb2d101836d04e46794/src/index.d.ts#L27) is defined.

should I do it inline?

Reply:
Sorry about that! I just realized you have two separate ones SocialMediaType and SocialIconType. It looked like the same when i took glance. 😄